apps/token-manager/contracts/TokenManager.sol

```diff
@@ -353,17 +353,8 @@ contract TokenManager is ITokenController, IForwarder, AragonApp {
 
         // vestedTokens = tokens * (time - start) / (vested - start)
         uint256 vestedTokens = SafeMath.div(
-            SafeMath.mul(
-                tokens,
-                SafeMath.sub(
-                    time,
-                    start
-                )
-            ),
-            SafeMath.sub(
-                vested,
-                start
-            )
+            tokens.mul(time.sub(start)),
+            vested.sub(start)
```

Comment:
Absolutely, I would even get rid of `SafeMath.div`. According to `assignVested` function it seems that `vested` could be equal to `start`, but I don't think that it makes much sense, and in case it does, we should treat it separately and explicitly like we do for other edge cases at the beginning of this function.

Reply:
This is not an issue as if `vested == start` then also `cliff == start`, for this reason, when `time < cliff` the shortcut returns `tokens` non vested tokens and when `time >= vested` it returns `0` not vested tokens, making it impossible for the function to ever try to divide by 0.

Reply:
So then we can remove de `SafeMath.div` then, which only advantage is now that it reverts instead of asserting. If you don't disagree, I'm pushing a commit removing it.